allensdk/brain_observatory/behavior/data_objects/timestamps/util.py

```diff
@@ -0,0 +1,5 @@
+import numpy as np
+
+
+def calc_frame_rate(timestamps: np.ndarray):
+    return np.round(1 / np.mean(np.diff(timestamps)), 0)
```

Comment:
I didn't see where this was calculated before, so I just wanted to check that we know timestamps are in the right units for this to work.

Reply:
I didn't make any changes to existing logic, so if it was working before, then it should still be working as expected